### PR TITLE
Piedram/appeals 59933

### PR DESCRIPTION
**Pull request by piedram**
Resolves [APPEALS-59933](https://jira.devops.va.gov/browse/APPEALS-59933)

# Description
Create Review Transcript task in the Hearing Admin queue.
1- Review Transcript task is created for each instance of
  - A PDF Transcript file being returned for a Hearing
  - A Word Document error file is returned for a Hearing and there is not a PDF Transcript
2-Review Transcript task is added to the Unassigned tab of the Hearing Admin team cases Queue

## Acceptance Criteria
- [x] Code compiles correctly

## Testing Plan
You will see the results when the job `monitor_box_job ` is executed

# Frontend
 To see the result go to Hearing admin case page

### CODE REVIEW

File: app/jobs/hearings/va_box_download_job.rb [hunk 1]
```diff
@@ -51,10 +50,40 @@ def update_database(current_information, file_status)
...
     end
+
+    modified_task_tree(current_information, file_status)
   rescue ActiveRecord::RecordInvalid => error
     Rails.logger.error "Failed to create transcription file: #{error.message}"
   end
 
+  def modified_task_tree(current_information, file_status)
+    if current_information["hearing_type"] == "Hearing"
+      current_hearing = Hearing.find(current_information["id"].to_i)
```
Comment (wmedders21) [inline app/jobs/hearings/va_box_download_job.rb:61]: unnecessary to convert to integer - ActiveRecord `.find` can accept strings.

File: app/jobs/hearings/va_box_download_job.rb [hunk 2]
```diff
@@ -64,15 +93,26 @@ def create_link_aws(current_information, file_status, transcript_text_path)
   end
 
   def create_transcription_file_record(current_information, file_status, aws_link)
-    transcription_id = current_information["hearing_type"].constantize.find(
-      current_information["id"]
-    ).transcription&.id
+    if current_information["hearing_type"] == "Hearing"
+      transcription_id = current_information["hearing_type"].constantize.find(
```
Comment (wmedders21) [inline app/jobs/hearings/va_box_download_job.rb:97]: If line 96 is true, you can use `Hearing.find`... on line 97. No need to call `.constantize` if you know the constant.

File: app/jobs/hearings/va_box_download_job.rb (hunk 1 above)
Comment (piedram) [reply app/jobs/hearings/va_box_download_job.rb:61]: Done!

File: app/jobs/hearings/va_box_download_job.rb (hunk 2 above)
Comment (piedram) [reply app/jobs/hearings/va_box_download_job.rb:97]: Done!

**Review by mchbidwell — COMMENTED** [review]
Looks to be a failed test for the Job
and a couple of Linting errors

**Review by mchbidwell — APPROVED** [review]

**Pull request merged**